tools/cmd/extend_oem/extend_oem_bin.go

```diff
@@ -0,0 +1,30 @@
+package main
```

Comment:
Any reason for naming the file: extend_oem_bin.go and not extend_oem.go?

Reply:
It might be confused with tools/extend_oem_partition.go.
The "_bin" shows that this Go program is used to generate a binary.